modules/nvidia-gpu-admin-dashboard-installing.adoc

```diff
@@ -0,0 +1,137 @@
+
+$ oc -n {{ .Release.Namespace }} get all -l app.kubernetes.io/name=console-plugin-nvidia-gpu
+
+Enable the plugin by running the following command:
+
+# Check if a plugins field is specified
+$ oc get consoles.operator.openshift.io cluster --output=jsonpath="{.spec.plugins}"
+
+# if not, then run the following command to enable the plugin
+$ oc patch consoles.operator.openshift.io cluster --patch '{ "spec": { "plugins": ["console-plugin-nvidia-gpu"] } }' --type=merge
+
+# if yes, then run the following command to enable the plugin
+$ oc patch consoles.operator.openshift.io cluster --patch '[{"op": "add", "path": "/spec/plugins/-", "value": "console-plugin-nvidia-gpu" }]' --type=json
+
+# add the required DCGM Exporter metrics ConfigMap to the existing NVIDIA operator ClusterPolicy CR:
+oc patch clusterpolicies.nvidia.com gpu-cluster-policy --patch '{ "spec": { "dcgmExporter": { "config": { "name": "console-plugin-nvidia-gpu" } } } }' --type=merge
+
+----
++
+The dashboard relies mostly on Prometheus metrics exposed by the NVIDIA DCGM Exporter, but the default exposed metrics are not enough for the dashboard to render the required gauges. Thus, the DGCM exporter should be configured to expose a custom set of metrics, as shown here.
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  dcgm-metrics.csv: |
+    DCGM_FI_PROF_GR_ENGINE_ACTIVE, gauge, gpu utilization.
+    DCGM_FI_DEV_MEM_COPY_UTIL, gauge, mem utilization.
+    DCGM_FI_DEV_ENC_UTIL, gauge, enc utilization.
+    DCGM_FI_DEV_DEC_UTIL, gauge, dec utilization.
+    DCGM_FI_DEV_POWER_USAGE, gauge, power usage.
+    DCGM_FI_DEV_POWER_MGMT_LIMIT_MAX, gauge, power mgmt limit.
+    DCGM_FI_DEV_GPU_TEMP, gauge, gpu temp.
+    DCGM_FI_DEV_SM_CLOCK, gauge, sm clock.
+    DCGM_FI_DEV_MAX_SM_CLOCK, gauge, max sm clock.
+    DCGM_FI_DEV_MEM_CLOCK, gauge, mem clock.
+    DCGM_FI_DEV_MAX_MEM_CLOCK, gauge, max mem clock.
+kind: ConfigMap
+metadata:
+  annotations:
+    meta.helm.sh/release-name: console-plugin-nvidia-gpu
+    meta.helm.sh/release-namespace: nvidia-gpu-operator
+  creationTimestamp: "2022-10-26T19:46:41Z"
+  labels:
+    app.kubernetes.io/component: console-plugin-nvidia-gpu
+    app.kubernetes.io/instance: console-plugin-nvidia-gpu
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: console-plugin-nvidia-gpu
+    app.kubernetes.io/part-of: console-plugin-nvidia-gpu
+    app.kubernetes.io/version: latest
+    helm.sh/chart: console-plugin-nvidia-gpu-0.2.3
+  name: console-plugin-nvidia-gpu
+  namespace: nvidia-gpu-operator
+  resourceVersion: "19096623"
+  uid: 96cdf700-dd27-437b-897d-5cbb1c255068
+----
++
+This is achieved by deploying this ConfigMap and editing the NVIDIA Operator ClusterPolicy CR to add that ConfigMap in the DCGM exporter configuration. The deployment of the ConfigMap is done by the new version of the Console Plugin NVIDIA GPU Helm Chart, but the ClusterPolicy CR editing should be done by the user.
```

Comment:
Recommend replacing "should be done" passive voice with "is"

Reply:
corrected.